openedx_external_enrollments/tests/external_enrollments/test_greenfig_external_enrollment.py

```diff
@@ -29,8 +29,11 @@ def test_get_enrollment_data(self, datetime_now_mock, _get_course_list_mock, get
         course_settings = {
             'external_course_run_id': 'course_id+10',
         }
-        expected_data = u'08-04-2020 10:50:34, John Doe, John, Doe, johndoe@email.com, course_id+10, true\n'
-        expected_data += u'08-04-2020 10:50:34, Mary Brown, Mary, Brown, marybrown@email.com, course_id+10, true\n'
+        dropbox_response_text = '08-04-2020 10:50:34, John Doe, John, Doe, johndoe@email.com, course_id+10, true\n'
+        expected_enrollment_data = '08-04-2020 10:50:34, Mary Brown, Mary, Brown, '\
+                                   'marybrown@email.com, course_id+10, true\n'
+        expected_file_data = dropbox_response_text
+        expected_file_data += expected_enrollment_data
```

Comment:
@Jacatove, can we keep this in a single assignment?
```suggestion
        expected_file_data = dropbox_response_text + expected_enrollment_data
```